.travis.yml

```diff
@@ -86,7 +86,7 @@ jobs:
       env: DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY=no EXTRA_ARGS=32_only
       # XXX i#2764: Travis OSX resources are over-subscribed and it can take
       # hours to get an OSX machine, so we skip running PR's for now.
-      if: type = push
+      # if: type = push
```

Comment:
(NOCHECKIN of course -- good practice to mark it with a comment so it's clearly temporary (we used to use #NOCHECKIN but CI now yells at that so something else))

Reply:
Ack